### PR TITLE
Show ae/wae image same everywhere

### LINKED ISSUE #358: Import pool ae icon should be the same as everywhere
image

### PR DESCRIPTION
**Pull request by CedrikNikita**
fixes #358

### CODE REVIEW

**Review by kenodressel — APPROVED** [review]

**Comment by kenodressel** [comment]
@bogdan-manole please review and merge if you are happy

**Comment by Liubov-crypto** [comment]
There is no URL for testing...

**Comment by Liubov-crypto** [comment]
Add liquidity AE and FST: 
add liquidity
not successful although I have tokens on my balance: 
add liquid 2

Swap AE to WAE, AE and WAE have the same icon:

swap ae to wae
ae to wae

Add liquidity AE Partner and AE: 

liquid

Looks like errors but all transactions were successful : 

errors

https://explorer.testnet.aeternity.io/transactions/th_zhVdMW2QZ6231xZB4Z7RQfikzsqYYzQ1pK99NxJhWkA9PQNid
https://explorer.testnet.aeternity.io/transactions/th_2Y4HWJhWkUZVP3fMn4S24Yf1MnCskQBvoKsxDCqRKB3YGenqkL
https://explorer.testnet.aeternity.io/transactions/th_XzTKbuAQFYL7QWk28J9wVihSwQkZ5Kwjanh5YfB1GjveHacsi
https://explorer.testnet.aeternity.io/transactions/th_vJPtxVzmLxkuLeeptKjAy5oBy8tzNmmNkNr7vLnrqVJygtz3n

1SND and 2 AE - result Revert: 
https://explorer.testnet.aeternity.io/transactions/th_2QVr4myACHZs4grQX4RW32m5wdSAKbUW1YBHKcWKpjL6iWnfCf
https://explorer.testnet.aeternity.io/transactions/th_2BEWCb1TjvqSuH1udWGZPDSUPG4Uy1eK3MACDiYXZCWtWTzWpS

Provide liquidity 1 AE and 7 FST, result OK: 
https://explorer.testnet.aeternity.io/transactions/th_Bhupvg9fqDpnJk4aPwPPGi5LsWzKiuRDmsNJnCkMQvJW6awPz 

Provide liquidity 1 WAE and 1 AE, result reverse:
https://explorer.testnet.aeternity.io/transactions/th_291LvUX8pQ3VCSdoHQqDxStW8ymzSSvk7bZysQkHqJhs9EKYMG

Swap 10 WAE for 10 AE, result ok: 
https://explorer.testnet.aeternity.io/transactions/th_XzTKbuAQFYL7QWk28J9wVihSwQkZ5Kwjanh5YfB1GjveHacsi

In poll I can choose a WAE token and create a pair from time to time: 
pool
wae ae pool

Different icons of tokens on modals: 

pools1
wae fst
wae fst

File: src/components/TokenIcon.vue [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+export default {
+  props: {
+    tokenA: { type: Object, required: true },
+    tokenB: { type: Object, default: null },
+    rotating: { type: Boolean },
+  },
+  methods: {
+    mapToken(token) {
+      return token.is_ae !== undefined ? AeIcon : `https://avatars.z52da5wt.xyz/${token.contract_id}`;
```
Comment (bogdan-manole) [inline src/components/TokenIcon.vue:27]: In my case, when used from PoolView, `token.is_ae` is always `null`

image

**Review by bogdan-manole — CHANGES_REQUESTED** [review]

File: src/components/TokenIcon.vue (hunk 1 above)
Comment (Liubov-crypto) [reply src/components/TokenIcon.vue:27]: Shouldn't tokens have the same appearance everywhere on all modals?

Comment (CedrikNikita) [reply src/components/TokenIcon.vue:27]: fixed

**Pull request merged**